src/traverse.ts

```
@@ -69,7 +69,9 @@
           return entry;
         }
 
-        const childUrl = new URL(entry.path, rootUrl.endsWith("/") ? rootUrl : `${rootUrl}/`).href;
+        const childUrl = rootUrl.endsWith("/") 
```

ESLint

[error] Trailing spaces not allowed.

(style/no-trailing-spaces)